### PR TITLE
update fault injection demo

### PR DESCRIPTION
**Pull request by devadvocado**

### CODE REVIEW

File: kubernetes/README.md [hunk 1]
```diff
@@ -1067,4 +1069,46 @@ http://192.168.64.62:30911
...
+    destinations:
+        - match:
+            service: backend.kuma-demo.svc:3001
+            protocol: http
+    conf:        
+        abort:
+            httpStatus: 500
+            percentage: 50
+        delay:
+            percentage: 99
```
Comment (lobkovilya) [inline kubernetes/README.md:1104]: I think we also can show that percentage might be fractional 99.99 for example

File: kubernetes/README.md [hunk 2]
```diff
@@ -1067,4 +1069,46 @@ http://192.168.64.62:30911
...
+apiVersion: kuma.io/v1alpha1
+kind: FaultInjection
+mesh: default
+metadata:
+  namespace: default
+  name: everything
+spec:
+    sources:
+        - match:
+            service: frontend.kuma-demo.svc:8080
```
Comment (lobkovilya) [inline kubernetes/README.md:1093]: Also probably worth mentioning, `service` isn't a mandatory tag in contrast to some other policies. The same matching could be done using  `app: kuma-demo-backend` and `app: kuma-demo-frontend`, which is a bit more user-friendly

File: kubernetes/kuma-demo-aio.yaml [hunk 3]
```diff
@@ -136,12 +136,14 @@ spec:
       app: kuma-demo-backend
       version: v0
       env: prod
+      protocol: http
```
Comment (jakubdyszkiewicz) [inline kubernetes/kuma-demo-aio.yaml:139]: This is not how we describe that service runs on HTTP.
https://kuma.io/docs/0.4.0/policies/http-support-in-kuma/

File: vagrant/README.md [hunk 4]
```diff
@@ -538,3 +540,41 @@ Visit the [Grafana dashboard](http://192.168.33.80:3000) to see the L4 metrics w
...
+conf:        
+    abort:
+        httpStatus: 500
+        percentage: 50
+    delay:
+        percentage: 99
+        value: 5s
+    responseBandwidth:
+        limit: 50 mbps
+        percentage: 50
```
Comment (jakubdyszkiewicz) [inline vagrant/README.md:574]: I'd remove `responseBandwidth` from demo. It's hard to showcase.

File: kubernetes/README.md (hunk 2 above)
Comment (jakubdyszkiewicz) [reply kubernetes/README.md:1093]: I'd go with `service` for now. It's consistent with all other stuff in the demo

**Comment by devadvocado** [comment]
merged in pr#71

**Pull request closed without merging**